scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -315,14 +384,9 @@ final case class BigQueryTypedTable[T: Coder](
 }
 
 /** Get an IO for a BigQuery table. */
+@deprecated("use BigQueryTypedTable(table, Format.TableRow) instead", "0.9.2")
 final case class BigQueryTable(table: Table) extends BigQueryIO[TableRow] {
```

Comment:
I suppose this could simply be a def and skip the class impl entirely ?
```suggestion
final def BigQueryTable(table: Table) = BigQueryTypedTable(table, Format.TableRow)
```

Reply:
Yeah we can ... we will loose bin-compat but i guess that's ok? we never strived for it in scio really right?

Reply:
Bin compat in Scio is IMHO an impossible goal bc. Beam or its dependencies happily break it with every release.

Reply:
One a second thought here ... I think it's a bit unlikely but this would be a possible breaking change if users have this IO in an assignment with explicit type.

Reply:
Might not be a big deal if this PR lands on v0.10